### PR TITLE
Create api routes for drop feature.

### LINKED ISSUE #4: API routes required for File Transfer
Need API routes for file transfer and drop access.
1. Open Drop
2. Create Drop
3. Encrypt file
4. Decrypt file

### PR DESCRIPTION
**Pull request by shubhansu-kr**
Changes: 

1. Added Figma design for client
2. Installed dependencies 
     - nanoid
     - crypto
3. Created drop db schema in file (./Server/models/drop.js)
4. Created Routes and Controller for Drop. 
     - Created file: (./Server/controller/dropController.js)
     - Created file: (./Server/route/dropRouter.js)

### CODE REVIEW

**Comment by shubhansu-kr** [comment]
@PatelNeelMahesh sir, Please review the changes.

File: Server/models/Drop.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+const dropSchema = new mongoose.Schema(
+    {
+        did: {
+            type: String,
```
Comment (PatelNeelMahesh) [inline Server/models/Drop.js:6]: Explore on usage of `Require` here.

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge PR.

File: Server/models/Drop.js (hunk 1 above)
Comment (shubhansu-kr) [reply Server/models/Drop.js:6]: > Explore on usage of `Require` here.

Sure, currently facing issue importing some libraries. Will be resolved in next PR.

ThankYou

**Pull request merged**